packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -88,13 +88,15 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
       return openCreateOrganization({ afterCreateOrganizationUrl, skipInvitationScreen });
     };
 
-    const handleManageOrganizationClicked = () => {
+    const handleItemClick = (path?: string) => {
       close();
       if (organizationProfileMode === 'navigation') {
         return navigateOrganizationProfile();
       }
+
       return openOrganizationProfile({
         ...organizationProfileProps,
+        ...(path && { path }),
```

Comment:
Shouldn't this also be 
```suggestion
        ...(__experimental_startPath && { __experimental_startPath }),
```